docs/en/ingest-management/beats-agent-comparison.asciidoc

```diff
@@ -238,6 +240,106 @@ per output type, which enables loadbalancing.
 |Supported
 |===
 
+[discrete]
```

Comment:
I'm assuming this is just a move operation with no changes to the text?

Reply:
@dedemorton I'm not sure what you mean? This is a whole new table. I just copied this line from the table above, but am I not supposed to?

Reply:
Ohhh. I need to slow down a little on reviews. I'll take a closer look.